lib/models/src/main/kotlin/no/nav/amt/lib/models/hendelse/HendelseDeltaker.kt

```diff
@@ -15,6 +15,7 @@ data class HendelseDeltaker(
         val navn: String,
         val arrangor: Arrangor,
         val tiltak: Tiltak,
+        val oppstart: Oppstartstype,
```

Comment:
Den må vel være nullable, eller ha en default, for å være bakoverkompatibel siden dette lagres i databasen i amt-distribusjon. Men det kan jo også fikses i amt-distribusjon.

Reply:
Vet ikke om jeg skjønte det helt. Sier du at dette er en dbo i amt-distribusjon?